### PR TITLE
Use CopyText on Git repo URLs.

### PR DESCRIPTION
**Pull request by adamjspooner**
@keybase/react-hackers, this is ready for review.

### CODE REVIEW

File: shared/git/row/index.js [hunk 1]
```diff
@@ -136,27 +145,9 @@ class Row extends React.Component<Props, State> {
                 }}
```
Comment (buoyad) [inline shared/git/row/index.js]: might need a `width` somewhere around here
image

Comment (adamjspooner) [reply shared/git/row/index.js]: Good catch! Thanks!

**Comment by cecileboucheron** [comment]
I think they could be wider on desktop - like could we have them be their maxWidth (should be set to 460px) and shrink if the window is smaller? It's nice to see the whole string if there is space for it. 

image

**Comment by adamjspooner** [comment]
@keybase/react-hackers, this is ready for another review. There are two problems related to the `CopyText` component on mobile:
1. The `Toast` component is covering the entire view on iOS making it impossible to tap anything.
1. The text is wrapping on Android (even with `numberOfLines={1}`) making it clipped on top and bottom.

**Comment by adamjspooner** [comment]
I can't review those changes, but they look good to me. Thanks, @buoyad!

**Comment by buoyad** [comment]
[My changes](https://github.com/keybase/client/pull/12246/files/09bc145898756c0107781468aedf7b619ef399ef..e21ce4f5fae4232fdae22bd4beca220ab390b5ca), if someone would like to look

File: shared/common-adapters/toast.native.js [hunk 2]
```diff
@@ -8,28 +8,40 @@ import type {Props} from './toast'
...
+        }).start()
+      )
     }
     if (!this.props.visible && prevProps.visible) {
       NativeAnimated.timing(this.state.opacity, {
         duration: 100,
         easing: NativeEasing.linear,
         toValue: 0,
       }).start()
+      this.timeoutID = setTimeout(() => this.setState({visible: false}), 100)
```
Comment (adamjspooner) [inline shared/common-adapters/toast.native.js]: Should the callback also `clearTimeout`?

File: shared/common-adapters/toast.native.js [hunk 3]
```diff
@@ -8,31 +8,43 @@ import type {Props} from './toast'
...
-        duration: 50,
+        duration: 100,
         easing: NativeEasing.linear,
         toValue: 0,
       }).start()
+      this.timeoutID = setTimeout(() => this.setState({visible: false}), 100)
     }
   }
 
+  componentWillUnmount() {
```
Comment (chrisnojima) [inline shared/common-adapters/toast.native.js]: we should use timerhoc and not do this outselves

Comment (buoyad) [reply shared/common-adapters/toast.native.js]: done 👍

File: shared/common-adapters/toast.native.js (hunk 2 above)
Comment (buoyad) [reply shared/common-adapters/toast.native.js]: I don't think it's necessary, by the time the callback happens the timer is already done

Comment (adamjspooner) [reply shared/common-adapters/toast.native.js]: Yeah, total nitpick.

**Comment by adamjspooner** [comment]
@keybase/react-hackers, bump.

**Pull request merged**